### PR TITLE
Fix/sinkdiv hessian

### LINKED ISSUE #320: `implicit` parameter not used in the Sinkhorn Divergence Hessians notebook?
Hi,

I'm interested in computing Hessians of OT costs, so I had a look at the Sinkhorn Divergence Hessians notebook.

Did I misunderstand something or is the `implicit` parameter in the `loss` function not used? Especially since the %time_it seems to be exactly the same in the unrolled version and in the implicit version.

Should the code not be as below?
```python
"implicit_diff": implicit_lib.ImplicitDiff(precondition_fun=lambda x: x) if implicit else None
```

Also, when should I use the default `precondition_fun = lambda x: eps*jnp.log(x)`, and when should I use `precondition_fun = lambda x: x`?

Finally, a very minor remark: at some point the text reads "Let’s parse the three lines in the call" but due to linting there are more than three lines. 

Thanks a lot!

### PR DESCRIPTION
**Pull request by michalk8**
closes #320

### CODE REVIEW

**Comment by marcocuturi** [comment]
Thanks Michal! LGTM, but it seems the plot is gone?
